doc/api.js

```diff
@@ -298,7 +298,21 @@ export function getDllVersion(callback) {}
  */
 export function getConnectedDevices(callback) {}
 
-
+/**
+ * Async function to get a list of all connected devices serialnumber.
```

Comment:
Perhaps reword to something like "... get the serial numbers of all connected devices"?